### ISSUE TITLE
Add power spectrum regularizer

### ISSUE THREAD

**Issue by behzadarbab** [behzadarbab/Machine-Learning-Imaging#1]
### Write the spectrum regularizer
- [ ] Measure the original power spectrum with uncertainties from the visibilities
- [ ] define the loss function for the regularizer
- [ ] Add it to the set of regularizers with the corresponding hyperparameter
